NitroxServer/GameLogic/Entities/EntityManager.cs

```diff
@@ -71,6 +71,10 @@ public void RegisterNewEntity(Entity entity)
         public void PickUpEntity(string guid)
         {
             entityData.RemoveEntity(guid);
+            if (entityData.GlobalRootEntitiesByGuid.ContainsKey(guid))
```

Comment:
Change this to `entityData.Remove(guid)`.

Reply:
Much better thanks @Measurity  , i shall add this in.